ci/jenkins/test.sh

```diff
@@ -621,6 +626,12 @@ function deliver_antrea {
             scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" flow-aggregator.tar jenkins@[${IP}]:${DEFAULT_WORKDIR}/flow-aggregator.tar
             ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; docker load -i ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; docker load -i ${DEFAULT_WORKDIR}/flow-aggregator.tar" || true
         done
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+            DOCKER_IMG_VERSION=${KIND_CLUSTER}
```

Comment:
There some other places in test.sh that suppose a single `:latest` tag for the images. For example: `function clean_antrea`, `function deliver_antrea` (`make flow-aggregator-image`, `./hack/build-antrea-linux-all.sh`). If you do not want to fix these places, I think the above line 630 is not needed, and we cannot make the job run in parallel in the same kind testbed.  We need to deploy multiple kind VMs and join them to Jenkins, and each kind testbed can have only one slot for running Jenkins job.

Reply:
Thanks for the comment, will make the changes in all places.